### PR TITLE
Added an example application with documentation

### PR DESCRIPTION
**Pull request by bubeck**

### CODE REVIEW

**Review by kisvegabor — COMMENTED** [review]
This is a wonderful idea! Thank you!

File: example/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+Example for lv_i18n
```
Comment (kisvegabor) [inline example/README.md:1]: Please mention the example in the main README file.

File: example/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+to extract new texts from your main.c and add it to the translation
+files. Please the edit translations/*.yml to see, that the new key was
+added and implement a proper translation for "en-GB" and "ru-RU".
+
+Then follow above to "make compile" and "make main" to compile the
+library and application and see the new key being translated.
+
+
+
+
```
Comment (kisvegabor) [inline example/README.md:71]: Note these extra lines

**Comment by bubeck** [comment]
Fixed all proposal/remarks. Thanks!

**Review by kisvegabor — APPROVED** [review]
Nice, thank you!

**Pull request merged**